subcommands/keys/tuf_updates_review.go

```diff
@@ -17,35 +18,61 @@ func init() {
 		Short: "Show the Factory's TUF root metadata",
 		Run:   doTufUpdatesReview,
 	}
-	review.Flags().BoolP("prod", "", false, "Show the production version")
+	review.Flags().BoolP("raw", "", false, "Show the raw root.json")
+	review.Flags().BoolP("prod", "", false, "Show the production root.json")
 	tufUpdatesCmd.AddCommand(review)
 }
 
 func doTufUpdatesReview(cmd *cobra.Command, args []string) {
 	factory := viper.GetString("factory")
+	showRaw, _ := cmd.Flags().GetBool("raw")
 	showProd, _ := cmd.Flags().GetBool("prod")
+	if showProd && !showRaw {
+		subcommands.DieNotNil(errors.New(
+			"If the flag 'prod' is set then the flag 'raw' must also be set",
+		))
+	}
 
 	updates, err := api.TufRootUpdatesGet(factory)
 	subcommands.DieNotNil(err)
 
-	var rootToShow, newCiRoot, newProdRoot *client.AtsTufRoot
-	_, newCiRoot = checkTufRootUpdatesStatus(updates, false)
-	if updates.Updated.ProdRoot != "" {
-		subcommands.DieNotNil(
-			json.Unmarshal([]byte(updates.Updated.ProdRoot), &newProdRoot), "Updated prod root",
-		)
-	}
-	if newProdRoot == nil {
-		fmt.Println("There are no staged TUF updates.")
-		return
-	}
+	// Other states than Started or Applying are rejected by checkTufRootUpdatesStatus.
+	_, newCiRoot := checkTufRootUpdatesStatus(updates, false)
 
-	if showProd {
-		rootToShow = newProdRoot
+	if showRaw {
+		var newProdRoot, rootToShow *client.AtsTufRoot
+		if updates.Updated.ProdRoot != "" {
+			subcommands.DieNotNil(
+				json.Unmarshal([]byte(updates.Updated.ProdRoot), &newProdRoot), "Updated prod root",
+			)
+		}
+		if newProdRoot == nil {
+			// No effective changes yet, but we know how the prod root would look like
+			newProdRoot = genProdTufRoot(newCiRoot)
```

Comment:
I think I see your point here - just show the actual "raw" content. But you can't apply this - so there's really nothing to review. I wonder if the old approach of saying "There are no staged TUF updates." would cause less confusion.

Reply:
I think the initial implementation was underthought.
There is something to show here: the version increment and the new changelog message.

In case of human readable format the user would see who created the transaction (even if there are no other changes yet).
I find that information valuable; regardless of the fact that it cannot be applied yet.